### ISSUE TITLE
Feature: Make Overhead camera respect InvertMouse setting

### ISSUE THREAD

**Issue by badosu** [beyond-all-reason/spring#745]
While the `InvertMouse` setting is helpful for the spring camera, it doesn't work for overhead camera.

With both cameras default mmb dragging to pan orientation reverse being opposite the functionality is a bit clunky, at least by having the option work one can start hope on standardizing camera orientations.

Even if camera orientations aren't standardized, games can at least have the option to mitigate the clunkiness of having to invert the sign of scroll speeds.

**Comment by dzosz** [comment]
looks like invertmouse is working for both cameras however in opposite ways. 
i agree on standardization but fixing this will make half of users unhappy as they will have to toggle this option to match previous behavior.

**Comment by badosu** [comment]
> looks like invertmouse is working for both cameras however in opposite ways.

Did you try `/set InvertMouse 1` with overhead cam? `/viewta`

**Comment by dzosz** [comment]
> Did you try `/set InvertMouse 1` with overhead cam?

yes, exactly this. I have a feeling players don't know that option (it's marked as dev in UI so it's not shown by default) and those that prefer inverted use TA cam and the others use Spring Cam

**Comment by badosu** [comment]
Thank you for checking, I must be crazy as I was pretty sure tested this in the past and overhead cam didn't respect `InvertMouse`.

Anyway, adjusted issue for standardization.
